packages/now-cli/src/commands/list.js

```diff
@@ -327,7 +327,7 @@ export default async function main(ctx) {
     ).replace(/^/gm, '  ')}\n\n`
   );
 
-  if (deployments.length === 20) {
+  if (pagination && deployments.length === 20) {
```

Comment:
We can prevent this type of error from happening in the future if we switch this file to typescript 😄